views/aria/attributes.go

```diff
@@ -24,14 +23,3 @@ func Attributes(helpText string, helpFieldID string) templ.Attributes {
 		return templ.Attributes{}
 	}
 }
-
-func stripHTML(text string) string {
-	result, err := html2text.FromString(text, html2text.Options{})
-	if err != nil {
-		log.Printf("Error while stripping HTML from '%s': %s\n", text, err.Error())
-
-		return text
-	}
-
-	return result
-}
```

Comment:
Moved this method (in an adapted version) to views/util to enable reuse. Could not add this to views/helpers.templ as that introduced circular references.